api/__init__.py

```diff
@@ -39,19 +49,27 @@
 def create_app():
     # Initial configurations:
     CORS(app)
-    app.config['SECRET_KEY'] = ''
-    
+    app.config['SECRET_KEY'] = os.environ.get("SECRET_KEY")
```

Comment:
What is the `SECRET_KEY` purpose? To prevent XSRF? If yes, when/how does it generated? By the `CORS(app)` call?

Reply:
Secret key needs to be set in order to use sessions

Reply:
For now I just have the user insert their own secret key in their end file

Reply:
If by you mean the server operator, that makes sense!